### PR TITLE
Fix go module path

### PR DESCRIPTION
**Pull request by Links2004**
fix module path (#560)
run go mod tidy, go fmt, and go vet

Signed-off-by: Markus Sattler <974709+Links2004@users.noreply.github.com>

### CODE REVIEW

**Comment by Links2004** [comment]
rebase and updated Makefile for new paths

File: opensearch-operator/controllers/suite_test_helpers.go [hunk 1]
```diff
@@ -95,11 +95,10 @@ func ArrayElementContains(array []string, content string) bool {
 }
 
 func ComposeOpensearchCrd(clusterName string, namespace string) opsterv1.OpenSearchCluster {
-
 	OpensearchCluster := &opsterv1.OpenSearchCluster{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "OpenSearchCluster",
-			APIVersion: "opensearch.opster.io/v1",
+			APIVersion: "github.com/Opster/opensearch-k8s-operator/opensearch-operator/v1",
```
Comment (swoehrl-mw) [inline opensearch-operator/controllers/suite_test_helpers.go:101]: Must stay `opensearch.opster.io` as this is the kubernetes apiversion and not the go package.

**Review by swoehrl-mw — CHANGES_REQUESTED** [review]
Hi @Links2004. Thank you for contributing this PR. Changes look mostly good, I found one place where too much was changed.

**Comment by swoehrl-mw** [comment]
@saketmht As you opened the releated issue, is this PR in line with what you need and would it work for you?

File: opensearch-operator/controllers/suite_test_helpers.go (hunk 1 above)
Comment (Links2004) [reply opensearch-operator/controllers/suite_test_helpers.go]: true, missed that one, updated the PR.

**Comment by saketmht** [comment]
> @saketmht As you opened the releated issue, is this PR in line with what you need and would it work for you?

Yes. This looks good!

**Comment by Links2004** [comment]
rebase and updated to resolve conflicts

**Review by swoehrl-mw — APPROVED** [review]

**Pull request merged**